UMLEditor/UMLEditor/Views/CommandLine.cs

```diff
@@ -46,13 +47,13 @@ private void ExecuteCommand(string input)
                     // Try to create a class with the first provided word as its name
                     _activeDiagram.AddClass(arguments.ElementAt(0));
 
-                    Console.WriteLine($"Class {words[1]} created");
+                    PrintColoredLine($"Class {words[1]} created", SUCCESS_COLOR);
                 }
 
                 else
                 {
-                    PrintColoredLine("To create a new Class, please enter a class name " +
-                                     "into the input box and then click press enter.", ERROR_COLOR);
+                    PrintColoredLine("To create a new Class, please enter \"add_class\" " +
+                                     "followed by a class name into the console and then click press enter.", ERROR_COLOR);

```

Comment:
The wording here is still a little strange. It says at the end "and then click press enter."